README.md

```diff
@@ -679,6 +679,17 @@ Offline Installations   | ✓     | x   | x   | x    |
   they could find ways to do an offline install for any package manager, but only `gomod` supports
   this out of the box (i.e. the user does not need to change their workflow).
 
+#### Limitations
+
+Cachito collects all dependencies from repository regardless of whether they used by a certain image
```

Comment:
This is not entirely accurate, Cachito collects dependencies for the packages that the user specifies (as you probably know, since you mention the multiple paths solution).

I'm also not sure if we should say "image" here - this is the first mention of container images in the README (apart from the containers that run Cachito itself). Cachito does not necessarily have to be used for building container images. Maybe "application," as used below, would be better.

Reply:
Used more precise wording:
```diff
- all dependencies
+ all accessible dependencies of specified package manager types
```

Replaced 'image' with 'application' in the first sentence. But in the second one it is an example, so any case specific term can be used.

Reply:
Can you clarify what you mean by "accessible"? And can we at least use "container image" instead of "image"? The word has too many meanings in the software world these days :D

Reply:
> Can you clarify what you mean by "accessible"?

Given the nature of the dependencies search, since we go through all subpaths and check availability of corresponding files, the word could be relevant.

> And can we at least use "container image" instead of "image"?

Agree. Let me fix it.

Reply:
Fixed 'images'.

Reply:
> > Can you clarify what you mean by "accessible"?
> 
> Given the nature of the dependencies search, since we go through all subpaths and check availability of corresponding files, the word could be relevant.

Hmm, I see. Maybe something like "Cachito collects all the dependencies defined in packaging files" would be more accurate? Not sure if "packaging files" is the right term, I tried to find a word that that would match npm lockfiles, pip requirements files and go.mod files at the same time.

Reply:
Removed 'accessible' to not confuse users.

Reply:
That works :+1: